### PR TITLE
Feature/export data to excell

### PR DESCRIPTION
**Pull request by idormenco**

### CODE REVIEW

**Comment by idormenco** [comment]
@bvizureanu , @RaduCStefanescu  can you check this on real db . Thanks a lot

File: src/VoteMonitor.Api.DataExport/Handlers/DataExportQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        public DataExportQueryHandler(VoteMonitorContext context, ILogger logger)
+        {
+            _context = context;
+            _logger = logger;
+        }
+
+        public async Task<List<ExportModel>> Handle(GetDataForExport request, CancellationToken cancellationToken)
+        {
+            var exportData = await _context.Answers
+                  .Where(a => a.IdObserver > 10)
```
Comment (RaduCStefanescu) [inline src/VoteMonitor.Api.DataExport/Handlers/DataExportQueryHandler.cs]: Usually we make a filtering for Ids that belong to internal testers, and they usually are dependent on the saved internal testers accounts.
This value is dynamic and it should be a parameter.

File: src/VoteMonitor.Api.DataExport/Handlers/DataExportQueryHandler.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                  .Where(a => a.LastModified >= new DateTime(2019, 11, 08, 6, 0, 0))
```
Comment (RaduCStefanescu) [inline src/VoteMonitor.Api.DataExport/Handlers/DataExportQueryHandler.cs]: Same comment here. The date depends on the date of elections and needs to be a parameter.

Comment (bvizureanu) [reply src/VoteMonitor.Api.DataExport/Handlers/DataExportQueryHandler.cs]: next step is parametrizing as per the issue description
filters on all fields

**Pull request merged**